packages/sveltekit/src/server/handleError.ts

```diff
@@ -41,9 +57,17 @@ export function handleErrorWithSentry(handleError: HandleServerError = defaultEr
  * so we have to check if the error is a "Not found" error by checking if the route id is missing and
  * by checking the error message on top of the raw stack trace.
  */
-function isNotFoundError(input: { error: unknown; event: RequestEvent }): boolean {
-  const { error, event } = input;
+function isNotFoundError(input: SafeHandleServerErrorInput): boolean {
+  const { error, event, status, message } = input;
+
+  // SvelteKit 2.0 offers a reliable way to check for a Not Found error:
+  if (status === 404 && message === 'Not Found') {
```

Comment:
l: is it just fine to only check status code?

Reply:
I added the check for both, message and status because they're not determined identically:

https://github.com/sveltejs/kit/blob/fc8389c9cf42a02665edd8aa7a6840fe982134c9/packages/kit/src/utils/error.js#L26-L39

So while I think it doesn't happen at the moment, a `HttpError` could be thrown with a 404 status code and another message which we'd also filter. If we only check for 404, we'd also filter out that error - whatever it is. Are we fine with that?

Reply:
I would assume any `404` errors are not useful, but perhaps it's good to be conservative on what we filter for now.

Reply:
I would not report 404s as errors. Filtering by transaction/span status should be powerful enough for users to achieve what they want.

Reply:
I changed the check to only consider the 404 status. You're right - we shouldn't capture other 404s either.